lib/Controller/Api1Controller.php

```diff
@@ -759,6 +764,13 @@ public function createColumn(
 		?string $selectionDefault = '',
 
 		?string $datetimeDefault = '',
+
+		?string $usergroupDefault = '',
+		?bool $usergroupMultipleItems,
+		?bool $usergroupSelectUsers,
+		?bool $usergroupSelectGroups,
+		?bool $showUserStatus,
```

Comment:
```suggestion
		?bool $usergroupMultipleItems = null,
		?bool $usergroupSelectUsers = null,
		?bool $usergroupSelectGroups = null,
		?bool $showUserStatus = null,
```

Need a default value as otherwise we have an error thrown:

> Optional parameter $usergroupDefault declared before required parameter $showUserStatus is implicitly treated as a required parameter